src/viewport.js

```diff
@@ -139,7 +143,14 @@ export class Viewport {
    * @return {!{width: number, height: number}}
    */
   getSize() {
-    return this.binding_.getSize();
+    var size = this.binding_.getSize();
```

Comment:
I'd much rather cache the whole "size" object and get rid of `this.width_` completely. And this method should cache the value as the case with others.